turbojpeg.py

```diff
@@ -227,6 +229,76 @@ def encode(self, img_array, quality=85, pixel_format=TJPF_BGR, jpeg_subsample=TJ
         finally:
             self.__destroy(handle)
 
+    """
+    @jpeg_buf           buffer containing jpeg image bytes
+    @pixel_format       pixel format of destination image
+    @jpeg_subsample     decoding format
```

Comment:
Remove `jpeg_subsample`.  Looks like we can't resample it.  It just has to be the same as the source image.